### PR TITLE
[docs][core] update quick starts with new dataset

### PR DESCRIPTION
**Pull request by ddhodge**
Tutorial for Explore Distributed SQL - new dataset featuring emp and dept tables, also used in Cloud Shell tutorial.

@netlify /latest/quick-start/explore/ysql/

### CODE REVIEW

**Review by aishwarya24 — COMMENTED** [review]
Added a comment , still need to review some of the pages.

File: docs/content/latest/yugabyte-cloud/cloud-faq.md [hunk 1]
```diff
@@ -71,6 +71,8 @@ If you want to continue testing YugabyteDB with more resource-intensive scenario
 - Download and run YugabyteDB on a local machine. For instructions, refer to [Quick Start](../../quick-start/).
 - Upgrade to a [standard cluster](../cloud-basics/create-clusters/) to access bigger clusters with more resources.
 
+To evaluate Yugabyte Cloud for production use or conduct a proof-of-concept (POC), contact {{<support-cloud>}} for trial credits.
+
 ### Can I migrate my free cluster to a standard cluster?
 
 Currently self-service migration is not supported. Contact [Yugabyte Support](https://support.yugabyte.com/hc/en-us/requests/new?ticket_form_id=360003113431) for help with migration.
```
Comment (aishwarya24) [inline docs/content/latest/yugabyte-cloud/cloud-faq.md:78]:
```suggestion
Currently self-service migration is not supported. Contact {{<support-cloud>}} for help with migration.
```

File: docs/content/latest/yugabyte-cloud/cloud-quickstart/_index.md [hunk 2]
```diff
@@ -15,7 +15,9 @@ menu:
 
 The quickest way to get started with YugabyteDB is to [sign up for Yugabyte Cloud](http://cloud.yugabyte.com) and create a free cluster. Although not suitable for production workloads, the free cluster includes enough resources to learn and develop applications with YugabyteDB.
 
-Get started with YugabyteDB on Yugabyte Cloud in a few steps:
+YugabyteDB is a distributed database optimized for deployment across a cluster of servers. The free cluster has a single node and limited resources, suitable for running tutorials, [Yugabyte University](https://university.yugabyte.com), and building sample applications. To evaluate Yugabyte Cloud for production use or conduct a proof-of-concept (POC), contact [Yugabyte Support](https://support.yugabyte.com/hc/en-us/requests/new?ticket_form_id=360003113431) for trial credits.
```
Comment (aishwarya24) [inline docs/content/latest/yugabyte-cloud/cloud-quickstart/_index.md:18]:
```suggestion
YugabyteDB is a distributed database optimized for deployment across a cluster of servers. The free cluster has a single node and limited resources, suitable for running tutorials, [Yugabyte University](https://university.yugabyte.com), and building sample applications. To evaluate Yugabyte Cloud for production use or conduct a proof-of-concept (POC), contact {{<support-cloud>}} for trial credits.
```

File: docs/content/latest/yugabyte-cloud/cloud-faq.md (hunk 1 above)
Comment (ddhodge) [reply docs/content/latest/yugabyte-cloud/cloud-faq.md:78]: Done in a previous PR

File: docs/content/latest/yugabyte-cloud/cloud-quickstart/_index.md (hunk 2 above)
Comment (ddhodge) [reply docs/content/latest/yugabyte-cloud/cloud-quickstart/_index.md:18]: Done in a previous PR

**Review by aishwarya24 — APPROVED** [review]
LGTM

**Review by aishwarya24 — APPROVED** [review]
LGTM

**Pull request merged**